scripts/lib/libvirt/admin-config

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+
+import libvirt
+import libvirt_helpers
+
+
+def main():
+    conn = libvirt.open("qemu:///system")
+    if not conn:
+        print("Failed to open connection to the hypervisor")
+        sys.exit(1)
+
+    parser = argparse.ArgumentParser(description="Create Admin Node Config.")
+    parser.add_argument("cloud", type=str, help="Name of the cloud")
+    parser.add_argument("adminnodememory", type=str, help="Amount of Memory")
+    parser.add_argument("adminvcpus", type=str, help="Number of VCPUs")
+    parser.add_argument("emulator", type=str, help="Emulator")
+    parser.add_argument("adminnodedisk", type=str,
+                        help="Disk of the Admin Node")
+    parser.add_argument("localrepositorymount", type=str,
+                        help="Local Repository Mount XML")
+
+    args = parser.parse_args()
+
+    cloud = args.cloud
+    cpuflags = libvirt_helpers.cpuflags()
+    fin = libvirt_helpers.readfile("templates/admin-node.xml")
```

Comment:
This is still not quite right.  The template name is not specific to the CLI code so it doesn't belong here.  The goal is ultimately to be able to write something like this from a Python-based `mkcloud` caller:

    import mkcloud.libvirt

    ...

    mkcloud.libvirt.write_admin_node_xml(output_file, values)